packages/frontend-2/components/viewer/resources/add-model/DialogObjectTab.vue

```diff
@@ -12,9 +12,9 @@
         label="Value"
         full-width
         size="lg"
-        :custom-icon="CubeIcon"
+        :custom-icon="LinkIcon"
         :rules="[isRequired, isValidValue]"
-        placeholder="Comma-delimited object IDs or an URL to an object(-s)"
+        placeholder="Comma-delimited object ID/URL"
```

Comment:
Should be plural, comma-delimiting only makes sense when there are multiple things to delimit